doc/run/cli.md

```diff
@@ -1,18 +1,24 @@
-# Run SQLFlow command-line tool
+# Run SQLFlow Command-line Tool
 
-We have a command-line tool called `sqlflow` which can access SQLFlow server like we do in Jupyter Notebook.  `sqlflow` works in the well known `REPL` mode, that is to say, reads SQL statements from the console, evaluates them by calling SQLFlow server, and prints the results.  `sqlflow` supports automatic code completion to ease applying the underlying powerful AI toolset of SQLFlow.
+In addition to the Jupyter Notebook magic command, we provide a command-line client `sqlflow` to connect and operate the SQLFlow gRPC server. Similar to other command-line client tools like `mysql`, `sqflow` implements several interaction modes.
+
+1. `sqlflow -e "SELECT ... TO TRAIN ..."` runs the SQLFlow program provided in the command line.
+1. `sqlflow -f a.sql` runs the SQLFlow program in a file.
+1. `sqflow` starts the REPL mode for user interaction.
+
+In the REPL mode, `sqlflow` supports automatic code completion and other features.
 
 ![](figures/repl.gif)
 
 ## Quick Start
 
-The SQLFlow Docker image contains the `sqlflow` client.  We can run an example session by typing the following command on MacOS.  If you run Docker on Linux, please change `host.docker.internal:3306` to `localhost:3306`.
+We can run an example session by typing the following command on macOS.  We assume you have installed [Docker](https://docs.docker.com/) on your computer.
```

Comment:
Change the link to `https://docs.docker.com/install/`?

Reply:
Done